src/charm.py

```diff
@@ -88,15 +93,11 @@ def func_with_catch_errors(self, event: EventT) -> None:
         except TokenError as err:
             logger.exception("Issue with GitHub token")
             self.unit.status = BlockedStatus(str(err))
-        except MissingConfigurationError as err:
```

Comment:
The `MissingConfigurationError` is now removed. Missing configuration will be managed by CharmState. The exception will eventually become a `ConfigurationError` now.